Common/Constants.cs

```diff
@@ -5,6 +5,22 @@ namespace Kudu
 {
     public static class Constants
     {
+        //Scan functionality files
+        public const string ScanLockFile = "scan.lock";
+        public static string ScanStatusFile = "status.json";
+        public static string ScanLogFile = "scan_log.log";
+        public static string ScanFolderName = "Scan_";
+        public static string MaxScans = "2";
+        public static string ScanDir = "/home/site/wwwroot";
+        public static string ScriptPath = "/custom_scripts/daily_scan_script.sh";
+        public static string ScanCommand = ScriptPath+" "+ScanDir;
+        public static string ScanTimeOut = "500000";
```

Comment:
Good to rename it to `ScanTimeoutInMilliSec` or add a comment that this value is in milliseconds

Reply:
Done.